src/main/java/org/jabref/gui/externalfiles/DroppedFileHandler.java

```diff
@@ -33,7 +33,6 @@
 import org.jabref.gui.undo.UndoableFieldChange;
 import org.jabref.gui.undo.UndoableInsertEntry;
 import org.jabref.logic.l10n.Localization;
-import org.jabref.logic.layout.LayoutFormatterPreferences;
```

Comment:
Same as above: Why is it save to remove the `LayoutFormatterPreferences`?

Reply:
See my e-mail reply below.